src/test/java/com/team3/central/services/CategoriesServiceTest.java

```diff
@@ -1,34 +1,43 @@
 package com.team3.central.services;
 
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
 import com.team3.central.repositories.CategoryRepository;
 import com.team3.central.repositories.entities.Category;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
-import static org.assertj.core.api.Assertions.assertThat;
 
 
 public class CategoriesServiceTest {
-  private static CategoryRepository mockCategoryRepository;
+
+  private static CategoryRepository categoryRepository;
   private static CategoriesService categoriesService;
 
+  private static Set<Category> getCategories() {
+    Set<Category> categories = Set.of(new Category("TestCategory1"), new Category("TestCategory2"));
+    return categories;
```

Comment:
you can do inline return (just do return Set.of(...)) . You don't have declare it